### ISSUE TITLE
Premium rules are not ignored when `null` and used without premium activated

### ISSUE THREAD

**Issue by SharakPL** [DEVSENSE/phptools-docs#522]
I use PSR12, which extends previous standards like PSR1 and PSR2:

> "php.format.codeStyle": "PSR-12",

So by default the extension should use this standard in formatting, but it looks like extension settings, even if not set, override chosen standard. First of all you might want to add `null` to acceptable value types to avoid warnings (unless it's a bug within VSCode itself):
![obraz](https://github.com/DEVSENSE/phptools-docs/assets/5007456/3f0e66fe-f093-4e80-a123-795b175d0dc8)

Even though I don't set any styling rules in settings.json (means this should be null and the rule directly from PSR12 standard should be used) and I don't even have premium active, these rules are used anyway like they were `true` instead of `null`:

![obraz](https://github.com/DEVSENSE/phptools-docs/assets/5007456/6dd6655c-54a0-4fe3-9abf-22a94ffc2645)

For example:
```php
if (isset($myVar)) {
    ...
}

if (!empty($myVar)) {
    ...
{
```
is formatted as 
```php
if (isset ($myVar)) {
    ...
}

if (!empty ($myVar)) {
    ...
{
```
It breaks rule [PSR2.Methods.FunctionCallSignature.SpaceBeforeOpenBracket](https://www.php-fig.org/psr/psr-2/#1-overview):

![obraz](https://github.com/DEVSENSE/phptools-docs/assets/5007456/b388513e-f29c-4b4c-b214-918fe5c55a81)

[These](https://www.php.net/manual/en/language.control-structures.php) are control structures:

- if
- else
- elseif/else if
- while
- do-while
- for
- foreach
- break
- continue
- switch
- match
- declare
- return
- require
- include
- require_once
- include_once
- goto

---
Of course I'm just guessing that's the rule causing this problem here. It would really help with debugging these if there was code style inspection available 🙏 Something like [PHP Sniffer](https://github.com/wongjn/vscode-php-sniffer) extension has:

![obraz](https://github.com/DEVSENSE/phptools-docs/assets/5007456/629e89d9-ff68-46ea-83db-3c86a3ef42a7)

showing specific rule that triggers styling error:

![obraz](https://github.com/DEVSENSE/phptools-docs/assets/5007456/64206e48-d1b9-4d9c-ae80-0fb90d9c2155)

**Comment by HB-2012** [comment]
Since the last update of the extension (1.45), I've noticed that the formatter in this extension automatically adds the space before opening parenthesis of the function calls. This was not the case before. I don't have premium and I do not have this setting enabled. Enabling or disabling suppressing premium features makes no difference. I do have some other PHP extensions but none that do formatting.

PHPCS then complains about it because like OP I've got PSR12 defined as coding standard.

I'm not sure what's going on, but since Google led me here after I experience the same warning and cause, I figured I'd share. There's no such thing as a coincidence 😄
